### PR TITLE
Bug fixes for issues #2242, #2243, #2245.

### PR DESCRIPTION
**Pull request by pankajrsingla**
Fixes https://github.com/jasp-stats/jasp-test-release/issues/2242
Fixes https://github.com/jasp-stats/jasp-test-release/issues/2243
Fixes https://github.com/jasp-stats/jasp-test-release/issues/2245

### CODE REVIEW

**Comment by vandenman** [comment]
@pankajrsingla if you start the pr message with
```
Fixes <link to issue>
```
then GitHub will automatically cross-link the issues under the "development" tab on the right.  In addition, once this PR is merged those issues will be automatically closed. I've edited your comment already to do this, but just so you know in the future.

File: R/commonFunctionsAcceptSampling.R [hunk 1]
```diff
@@ -451,7 +451,7 @@ getAOQCurve <- function(jaspContainer, pos, depend_vars, df_plan, options, type,
   xBreaks <- jaspGraphs::getPrettyAxisBreaks(c(min(df_plan$PD), max(df_plan$PD)))
   yBreaks <- jaspGraphs::getPrettyAxisBreaks(c(min(df_plan$AOQ), 1.2*aoql))
   plt <- ggplot2::ggplot(data = df_plan, ggplot2::aes(x = PD, y = AOQ)) +
-         ggplot2::geom_point(colour = "black", shape = 19) + ggplot2::labs(x = "(Incoming) Proportion non-conforming", y = "Average Outgoing Quality") +
+         ggplot2::geom_point(colour = "black", shape = 19) + ggplot2::labs(x = "(Incoming) Proportion Non-conforming", y = "Average Outgoing Quality") +
```
Comment (vandenman) [inline R/commonFunctionsAcceptSampling.R:454]: so this is fine but these also need
```suggestion
         ggplot2::geom_point(colour = "black", shape = 19) + ggplot2::labs(x = gettext("(Incoming) Proportion Non-conforming"), y = gettext("Average Outgoing Quality")) +
```
same for the other plot labels.

**Review by vandenman — CHANGES_REQUESTED** [review]

**Comment by pankajrsingla** [comment]
> @pankajrsingla if you start the pr message with
> 
> ```
> Fixes <link to issue>
> ```
> 
> then GitHub will automatically cross-link the issues under the "development" tab on the right. In addition, once this PR is merged those issues will be automatically closed. I've edited your comment already to do this, but just so you know in the future.

Thanks, will follow this going forward! :)

File: R/commonFunctionsAcceptSampling.R (hunk 1 above)
Comment (pankajrsingla) [reply R/commonFunctionsAcceptSampling.R]: Added it to the labels. Thanks for pointing out!

**Pull request merged**